charts/validator/templates/deployment.yaml

```diff
@@ -33,12 +33,28 @@ spec:
           hostNetwork: true
           {{- end }}
           image: "{{ .Values.ghost.image.repository }}:{{ .Values.ghost.image.tag | default .Chart.AppVersion }}"
+          {{- if .Values.ghost.commandOverride }}
+          command:
+          {{- range $key, $val := .Values.ghost.commandOverride }}
+            - {{ $val }}
+          {{- end }}
+          {{- end }}
           args:
+          {{- if .Values.ghost.argsOverride }}
+          {{- range $key, $val := .Values.ghost.argsOverride }}
+            - {{ $val }}
+          {{- end }}
+          {{- else }}
             - "run"
             - "-v"
             - "{{ .Values.ghost.logLevel | default "warning" }}"
             - "--log.format"
             - "{{ .Values.ghost.logFormat | default "text" }}"
+          {{- end }}
+            {{- if .Values.ghost.configsUrl }}
```

Comment:
if `configsUrl` is provided, it will set the flags, if not, it will use the default arguments as above

Reply:
actually, we can live without this, as we do not control configs by args (and we can override them) - but we use config registry